_includes/samples/APIRequests/APIConsumer.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Update this file with the changes from #313 and I think we're good to merge.

https://github.com/smartcontractkit/documentation/pull/313/files